### DISCUSSION TITLE
Rename GNOME-Colors to Gnome-Colors?

### DISCUSSION THREAD

**Discussion by actionless** [themix-project/themix-gui#348] [z Recycle Bin]
cc @nana-4 

I'm starting to have an impression, what the name of that palette doesn't reference to gnomes as in G.N.O.M.E.:

![2020-12-24--1608831351_1544x517_scrot](https://user-images.githubusercontent.com/1655669/103101300-3a39f100-4617-11eb-8f9f-b78b3ca936d0.png)

Also if it even mean a reference to GNOME project, nowadays they're using totally different palettes from this style, so it doesn't feel intuitive

**Comment by nana-4** [comment]
My apologies for the very late response :/

Just to be sure, are you talking about the label name in the sidebar?

I agree that it's misleading, but IMHO I don't think renaming `GNOME-Colors` to `Gnome-Colors` would help much. I'd suggest either:

- Rename it to `Shiki` (because they all have the `shiki` prefix)
- Rename it to `Color sets from GNOME-Colors` (to make it clear they from the icon theme called "GNOME-Colors")

**Comment by actionless** [comment]
> Rename it to Shiki (because they all have the shiki prefix)

yup, thanks for your feedback, i think that gonna be the most logical here

**Comment by actionless** [comment]
it seems to be not possible to close the github "discussion", only to remove it

**Reply by actionless** [reply]
i think i finally found a workaround -- creating "Recycle Bin" category to move "closed" discussions there
